invenio_curations/notifications/builders.py

```diff
@@ -21,79 +29,99 @@
 class CurationRequestActionNotificationBuilder(NotificationBuilder):
     """Notification builder for curation actions."""
 
-    type = "curation-request"
-
-    @classmethod
-    def build(cls, identity, request):
-        """Build notification with request context."""
-        return Notification(
-            type=cls.type,
-            context={
-                "executing_user": EntityResolverRegistry.reference_identity(identity),
-                "request": EntityResolverRegistry.reference_entity(request),
-            },
-        )
+    type: ClassVar[str] = "curation-request"
```

Comment:
is this class variables meant to be immutable or can it be mutated?

i read the rule from ruff in a way that only class variables with intention to be updated or changed should be marked as `ClassVar` and class variables which are immutable should be marked with `Final`

https://docs.astral.sh/ruff/rules/mutable-class-default/
https://docs.python.org/3/library/typing.html#typing.ClassVar
https://docs.python.org/3/library/typing.html#typing.Final

Reply:
yes i saw that too, but these class variables are overridden by subclasses (e.g., CurationRequestSubmitNotificationBuilder overrides both `type` and `recipients`), so ClassVar is correct in my opinion - final would prevent inheritance.

Reply:
if we used `Final` would prevent overriding in subclasses